### PR TITLE
Bump php and dependencies

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because this change should respect BC and is very important to feature updates.

## Changelog

```markdown
### Removed
- remove support for `jms/di-extra-bundle`
```

### CODE REVIEW

**Comment by wbloszyk** [comment]
@sonata-project/contributors RTM. This PR is very important for `make CoreBundle optional` in BlockBundle.

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ branches:
 language: php
 
 php:
-  - '7.1'
```
Comment (core23) [inline .travis.yml:13]: You need to update the dev-kit too, otherwise this will be reverted with the next cron run

Comment (jordisala1991) [reply .travis.yml:13]: The 3.x branch is not on dev-kit anymore: https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml#L105

**Comment by greg0ire** [comment]
Thanks @wbloszyk !

**Pull request merged**